week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -22,25 +35,39 @@ function greaterThan10() {
   }
 }
```

Comment:
You have deleted the else statement here

Reply:
it is there